src/django/api/helpers.py

```diff
@@ -61,3 +64,41 @@ def get_list_contributor_field_values(item, fields):
             index = list_fields.index(f['column_name'])
             f['value'] = data_values[index]
     return fields
+
+
+def clean(column):
```

Comment:
Moved this to the `helpers` file to avoid circular dependencies when using this in the `models` file

Reply:
@TaiWilkin Same! https://github.com/open-apparel-registry/open-apparel-registry/pull/1639#discussion_r804803932